core/templates/pages/exploration-editor-page/editor-tab/state-version-history/state-version-history.component.ts

```diff
@@ -41,7 +41,7 @@ export class StateVersionHistoryComponent {
     );
   }
 
-  getLastEditedVersionNumber(): number {
+  getLastEditedVersionNumber(): number | null {
```

Comment:
In what circumstance can it be null? You should have a docstring to explain.

Reply:
done